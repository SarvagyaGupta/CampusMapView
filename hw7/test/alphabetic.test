# Test correct printing of graph

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 b
AddNode graph1 a

# Add an edge
AddEdge graph1 a b 1
AddEdge graph1 b b 2
AddEdge graph1 a b 2
AddEdge graph1 b a 1
AddEdge graph1 a a 1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 a
ListChildren graph1 b
