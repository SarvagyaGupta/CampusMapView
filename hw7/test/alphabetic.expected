# Test correct printing of graph

# Create a graph
created graph graph1

# Add a pair of nodes
added node b to graph1
added node a to graph1

# Add an edge
added edge 1.000 from a to b in graph1
added edge 2.000 from b to b in graph1
added edge 2.000 from a to b in graph1
added edge 1.000 from b to a in graph1
added edge 1.000 from a to a in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: a b
the children of a in graph1 are: a(1.0) b(1.0) b(2.0)
the children of b in graph1 are: a(1.0) b(2.0)
